frontend/src/components/layout/AppNavBar/AppNavBar.tsx

```diff
@@ -40,6 +37,19 @@ function AppNavBar() {
   );
 }
 
+function HomeButton() {
```

Comment:
I generally lean towards separate files for components, however when the component isn't used anywhere the rule gets blurry.

Reply:
I also lean towards "one-file one-component" rule. However, I noticed this file had multiple components in it so I did not want to break the pattern there.

Can be refactored later for sure. Extracting all sub-components into separate files would be trivial